index.bs

```diff
@@ -838,12 +861,17 @@ When an {{XRReferenceSpace}} is requested, the user agent MUST <dfn>create a ref
   1. Let |session| be the {{XRSession}} object that requested creation of a reference space.
   1. Let |type| be set to the {{XRReferenceSpaceType}} passed to {{requestReferenceSpace()}}.
   1. If the [=reference space is supported=] for |type| and |session|, run the following steps:
-    1. If |type| is {{bounded-floor}}, let |referenceSpace| be a new {{XRBoundedReferenceSpace}}.
-    1. Else let |referenceSpace| be a new {{XRReferenceSpace}}.
+    1. Initialize |referenceSpace| based on the following:
+      <dl class="switch">
+        <dt> If |type| is {{bounded-floor}}
+        <dd> Let |referenceSpace| be a new {{XRBoundedReferenceSpace}}.
+        <dt> Else
```

Comment:
There are six instances of `Else`. Should those be `Otherwise` for consistency?

Reply:
Honsetly I think `Else` is fine in these cases, especially if it's just a simple `if/else` statement. Where I would expect `Otherwise` to apply is mostly at the end of a switch with several branches. (Functioning as `default` in a C++ switch, essentially)

Reply:
It seems like other specs use `Otherwise` with this most of the time (as @ddorwin said above)

https://drafts.csswg.org/css-transforms-2/#serialization-of-the-computed-value , https://drafts.csswg.org/web-animations-2/#the-intrinsic-iteration-duration-of-a-group-effect 
https://html.spec.whatwg.org/multipage/dom.html#directionality-of-the-attribute